php/admin_page.php

```diff
@@ -10,50 +38,61 @@
-        <div class="container">
+    <div class="mainText container">
             <h4>Welcome</h4>
             <h2>RICHARD</h2>
             <h3>'ABOUT ME'</h3>
-        </div>
     </div>
-
     <div class="forms container">
         <div class="formSize">
             <p>Add additional paragraphs</p>
-            <form method="post" action="admin_add.php" class="formAdd">
-                <textarea rows="6" cols="80" name="add_form" required></textarea>
+            <form method="post" action="admin_add.php" class="formAdd editParaRight">
+                <textarea name="add_form"  required></textarea>
+                <div>
                 <input type="submit" name="addSubmit">
+                </div>
             </form>
         </div>
         <div class="formSize">
             <p>Edit paragraph</p>
-            <form>
-                <textarea rows="6" cols="80" name="edit_form"></textarea>
+            <form method="post" class="formText" action="admin_page.php">
                 <h4>Choose which paragraph to edit</h4>
                 <select name="editSelect">
-                    <option>paragraph 1</option>
-                    <option>paragraph 2</option>
-                    <option>paragraph 3</option>
+                    <option selected="selected" value="choose">Choose Paragraph</option>
+                    <?php if(isset($editDropdownResults)){
+                        echo $editDropdownResults;
+                    } ?>
                 </select>
+                <div class="edit">
+                    <input type="submit" name="edit" value="Edit">
+                </div>
+            </form>
+            <form method="post" class="formText editParaRight" action="admin_page.php">
+                <textarea class="edit_submit" name="edit_form" required><?php
+                    if(isset($pasteEdit)){
+                        echo $pasteEdit;
+                    } ?></textarea>
+                <?php if (isset($editChoice)){
+                    echo '<input type="hidden" name="editId" value=' . $editChoice . ' />';
+                }
+                ?>
                 <div class="submit_buttons">
-                    <input type="submit" name="textEdit" value="Edit Text">
-                    <input type="submit" name="textSubmit" value="Submit New Text">
+                    <?php $submitButton = submitButton(); ?>
                 </div>
             </form>
         </div>
         <div class="formSize">
             <p>Delete paragraph</p>
-            <form>
+            <form method="post" action="admin_page.php">
                 <h4>Choose which paragraph to delete</h4>
                 <select name="deleteSelect">
-                    <option>paragraph 1</option>
-                    <option>paragraph 2</option>
-                    <option>paragraph 3</option>
+                    <option selected="selected" value="choose">Choose Paragraph</option>
+                    <?php if(isset($editDropdownResults)){
```

Comment:
This one too!

Reply:
no need for this one as dropdown is always present when page loads, removed the isset